assets/src/components/vehiclePropertiesPanel.tsx

```diff
@@ -20,7 +21,14 @@ const Header = ({
   hideMe: () => void
 }) => (
   <div className="m-vehicle-properties-panel__header">
-    <div className="m-vehicle-properties-panel__label">{vehicle.label}</div>
+    <div className="m-vehicle-properties-panel__label">
+      <VehicleIcon
+        size={Size.Large}
+        orientation={Orientation.Up}
+        label={vehicle.label}
```

Comment:
It feels like the logic of formatting the label and variant is kind of split between the callers and the component (e.g. the component overrides with a `variant !== "_"` check). What would you think about passing in a vehicle (so the component handles what it means to dispaly a label and variant based on the data), and then a `showLabel` boolean flag or something like that?

Reply:
Whether or not that is a good idea, one thing I just thought about that might play into this consideration is that we'll soon be coloring the icon based on the vehicle status (https://app.asana.com/0/1112935048846093/1122642346801331)

Reply:
Yeah sure. Everywhere that uses the icon has a vehicle associated with it, and all of them draw the variant. inside.

Reply:
Except ghost buses maybe? Will drawing them become much harder later if we make this assumption that you always have a vehicle?

Reply:
That's good to consider. The component could always accept an optional vehicle and handle it.

Reply:
There will also be `SW-OFF` vehicles where the label used isn't the vehicle id or the run, and pending box vehicles where we're not sure how we'll want to handle the status. Until we understand better how all of these will work, I'd lean towards continuing to have the callers decide what fields to pass in, and we can keep this idea in mind for the future.

Reply:
Interesting, yeah ok.